debezium-server-name-mapper/src/main/resources/application.properties

```diff
@@ -0,0 +1,18 @@
+debezium.sink.type=pulsar
+debezium.sink.pulsar.client.serviceUrl=pulsar://localhost:6650
+debezium.source.connector.class=io.debezium.connector.postgresql.PostgresConnector
+debezium.source.offset.storage.file.filename=target/offsets.dat
+debezium.source.offset.flush.interval.ms=0
+debezium.source.database.hostname=localhost
+debezium.source.database.port=5432
+debezium.source.database.user=postgres
+debezium.source.database.password=postgres
+debezium.source.database.dbname=postgres
+debezium.source.database.server.name=tutorial
+debezium.source.schema.whitelist=inventory
+
+# Needed till https://issues.redhat.com/browse/DBZ-2192 is released
```

Comment:
That's merged now (also answers my question on the issue :-). Can you remove this and rebase?

Reply:
No, I can't. Will remove only after the JARs with fix are released.